src/test/java/io/jenkins/plugins/checks/github/GitHubChecksPublisherITest.java

```diff
@@ -217,48 +227,50 @@ public void shouldPublishGitHubCheckRunCorrectly() {
     @Issue("issue-20")
     @Test
     public void shouldLogChecksParametersIfExceptionHappensWhenPublishChecks() {
-        loggerRule.record(GitHubChecksPublisher.class.getName(), Level.WARNING).capture(1);
-
-        ChecksDetails details = new ChecksDetailsBuilder()
-                .withName("Jenkins")
-                .withStatus(ChecksStatus.COMPLETED)
-                .withConclusion(ChecksConclusion.SUCCESS)
-                .withOutput(new ChecksOutputBuilder()
-                        .withTitle("Jenkins Check")
-                        .withSummary("# A Successful Build")
-                        .withAnnotations(Collections.singletonList(
-                                new ChecksAnnotationBuilder()
-                                        .withPath("Jenkinsfile")
-                                        .withStartLine(1)
-                                        .withEndLine(2)
-                                        .withStartColumn(0)
-                                        .withEndColumn(20)
-                                        .withAnnotationLevel(ChecksAnnotationLevel.WARNING)
-                                        .withMessage("say hello to Jenkins")
-                                        .build()))
-                        .build())
-                .build();
-
-        new GitHubChecksPublisher(contextBuilder.apply(this),
-                new PluginLogger(j.createTaskListener().getLogger(), "GitHub Checks"),
-                wireMockRule.baseUrl())
-                .publish(details);
-
-        assertThat(loggerRule.getRecords().size()).isEqualTo(1);
-        assertThat(loggerRule.getMessages().get(0))
-                .contains("Failed Publishing GitHub checks: ")
-                .contains("name='Jenkins'")
-                .contains("status=COMPLETED")
-                .contains("conclusion=SUCCESS")
-                .contains("title='Jenkins Check'")
-                .contains("summary='# A Successful Build'")
-                .contains("path='Jenkinsfile'")
-                .contains("startLine=1")
-                .contains("endLine=2")
-                .contains("startColumn=0")
-                .contains("endColumn=20")
-                .contains("annotationLevel=WARNING")
-                .contains("message='say hello to Jenkins'");
+        try (var credentialsMatchers = mockCredentialsMatchers()) {
```

Comment:
Introduced `try-with-resource` for static mock auto-close. The body is indented without any change.